dali/pipeline/operators/reader/loader/recordio_loader.h

```diff
@@ -94,6 +94,7 @@ class RecordIOLoader : public IndexedFileLoader {
       tensor.set_type(TypeInfo::Create<uint8_t>());
       tensor.Resize({1});
       tensor.SetSkipSample(true);
+      current_file_->Seek(seek_pos + size);
```

Comment:
What if you seek past the end of file? Please check L125 - it seems that image can be splitted between files, or even be in the next file but you won't know that until you try to read file and get n_read  == 0.

Reply:
done